src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -336,6 +367,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('default_share_permission', default_share_permission_type)
         c.argument('enable_nfs_v3', arg_type=get_three_state_flag(), is_preview=True, min_api='2021-01-01',
                    help='NFS 3.0 protocol support enabled if sets to true.')
+        c.argument('enable_vlw', arg_type=enable_vlw_type, validator=validate_immutability_arguments)
+        c.argument('immutability_period_since_creation_in_days',
+                   arg_type=immutability_period_since_creation_in_days_type,
+                   validator=validate_immutability_arguments)
+        c.argument('immutability_policy_state', arg_type=immutability_policy_state_type,
+                   validator=validate_immutability_arguments)
+        c.argument('allow_protected_append_writes', arg_type=allow_protected_append_writes_type,
```

Comment:
I'll suggest to all `arg_group` for all these parameters since `az storage account create|update` has so many parameters, it need to be grouped orderly.

Reply:
Got it.